right/src/macros.c

```diff
@@ -460,9 +460,30 @@ static macro_result_t processScrollMouseAction(void)
 static macro_result_t processClearStatusCommand()
 {
     statusBufferLen = 0;
+    SegmentDisplay_DeactivateSlot(SegmentDisplaySlot_Error);
+    SegmentDisplay_DeactivateSlot(SegmentDisplaySlot_Warn);
     return MacroResult_Finished;
 }
 
+
+char Macros_ConsumeStatusChar()
+{
+    static uint16_t readingPos = 0;
+    char res;
+
+    if (readingPos < statusBufferLen) {
+        res = statusBuffer[readingPos++];
+    } else {
+        res = '\0';
+        statusBufferLen = 0;
+        readingPos = 0;
+        SegmentDisplay_DeactivateSlot(SegmentDisplaySlot_Error);
```

Comment:
We should keep showing the error/warning status on the LED display because Agent may not be in the foreground when the error triggers and it reads the error, so the user may miss the error.